package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "karlsfriendshipregain",
+  "version": "1.0.0",
+  "description": "https://tohil.org",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thornjad/karlsfriendshipregain.git"
+  },
+  "author": "",
+  "license": "ISC",
```

ISC, MIT and BSD are almost identical, I made an arbitrary choice

```suggestion
  "license": "MIT",
```